.github/workflows/issue-greetings.yml

```diff
@@ -0,0 +1,21 @@
+name: Issue greetings
+
+on: [issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            👋 Hello @${{ github.actor }}, thank you for leaving an issue on Roboflow Notebooks.
+
+            ## 🐞 Bug reports
+
+            If you are filing a bug report, please be as detailed as possible. This will help us more easily diagnose and resolve the problem you are facing.
```

Comment:
I'd also add a link to [CONTRIBUTING.md](https://github.com/roboflow/notebooks/blob/main/CONTRIBUTING.md). @mo-traor3-ai is currently working on some `Bug report` and `Feature request` instructions there.